README.md

```diff
@@ -0,0 +1,138 @@
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Hello Microverse\] ](#-hello-microverse-)
+  - [🛠 Built With **HTML and CSS**](#-built-with-html-and-css)
+    - [Tech Stack **HTML and CSS**](#tech-stack-html-and-css)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+**Hello Microverse** is a project under microverse Module 1 block ! aimed at testing my knowledge about the concepts learnt in the first 2 days of the program.
+
+## 🛠 Built With **HTML and CSS**
+
+### Tech Stack **HTML and CSS**
+### Key Features <a name="key-features"></a>
+- **Displays Hello Microverse**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: A web browser.
+### Setup
+
+Clone this repository to your desired folder:
+cd Module_1
+-git clone git@github.com:As1imwe-Mark/HelloWorld.git
+
+### Install
+
+Install this project with:
+_No installation is required, double click the index.html file to open it.
+
+### Usage
+
+To run the project, execute the following command:
+-Double click the index.html file.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Asiimwe Mark**
+
+- GitHub: @As1imwe-Mark
+- Twitter:@markasiimwe
+- LinkedIn:AsiimweMark
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project like it and give me a follow.
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank Naan Ahmed [@naanahmed]👤 **Naan Ahmed**
+
+- GitHub: [@naanahmed](https://github.com/naanahmed) for his tireless support throughout the course of this project.
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] As stated by the previous reviewer, kindly delete this section of the README file. You can see its there as a guide, and should not be ideally added which makes your code professional 👍